source/includes/example-connect-onprem-atlas.rst

```diff
@@ -0,0 +1,50 @@
+Gather Connection Information
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The source cluster, ``cluster0``, is hosted on the following servers
+and ports:
+
+- clusterOne01.fancyCorp.com:20020
+- clusterOne02.fancyCorp.com:20020
+- clusterOne03.fancyCorp.com:20020
+
+The destination cluster, ``cluster1``, is hosted on the following
+servers and ports:
+
+- clusterTwo01.mongodb.net:27017
+- clusterTwo02.mongodb.net:27017
+- clusterTwo03.mongodb.net:27017
```

Comment:
Format should be something like this:

- clusterTwo-shard-00-00.abc12.mongodb.net:27017
- clusterTwo-shard-00-02.abc12.mongodb.net:27017
- clusterTwo-shard-00-01.abc12.mongodb.net:27017

Reply:
Ah gotcha. Wasn't sure if it was a matter of the `mongodb.net` or the full format. Thanks for the note.